packages/contract-wrappers/src/contract_wrappers/dutch_auction_wrapper.ts

```diff
@@ -129,22 +131,8 @@ export class DutchAuctionWrapper extends ContractWrapper {
     public async getAuctionDetailsAsync(sellOrder: SignedOrder): Promise<DutchAuctionDetails> {
         // type assertions
         assert.doesConformToSchema('sellOrder', sellOrder, schemas.signedOrderSchema);
-        // get contract
-        const dutchAuctionInstance = await this._getDutchAuctionContractAsync();
         // call contract
-        const auctionDetails = await dutchAuctionInstance.getAuctionDetails.callAsync(sellOrder);
+        const auctionDetails = await this._dutchAuctionContract.getAuctionDetails.callAsync(sellOrder);
         return auctionDetails;
     }
-    private async _getDutchAuctionContractAsync(): Promise<DutchAuctionContract> {
```

Comment:
I removed these `get__ContractAsync` methods everywhere for consistency. I think they are leftover from when instantiating `__Contract` involved something asynchronous. Now they can be instantiated directly in the constructor synchronously.